rules/limitdatetime/lang/en/registrationrule_limitdatetime.php

```diff
@@ -28,5 +28,6 @@
 
  $string['plugindescription'] = 'Only allow registration between two dates';
 $string['pluginname'] = 'Limit by date';
+$string['privacy:null_provider:reason'] = 'Limit by date rule simply limits registration by date. There\'s no user related data involved, stored of processed by the plugin.';
```

Comment:
Maybe... "There is no user related data involved, stored, or processed by the plugin." <- added a comma, corrected a typo etc

Reply:
Applied your suggestion!